src/import/pprof.ts

```diff
@@ -81,7 +81,13 @@ export function importAsPprofProfile(rawProfile: ArrayBuffer): Profile | null {
     if (lastLine == null) return null
 
     if (lastLine.functionId) {
-      return frameInfoByFunctionID.get(i32(lastLine.functionId)) || null
+      let funcFrame = frameInfoByFunctionID.get(i32(lastLine.functionId))
+      if (lastLine.line && lastLine.line > 0) {
+        if (funcFrame != null) {
+          funcFrame.line = lastLine.line as number
+        }
+      }
+      return funcFrame || null
```

Comment:
I took a closer look at this. I'm not sure why browser & node behavior is different, but the type signatures do indicate it's more correct here to handle the case that `lastLine.line` is an instance of `Long` instead of assuming it's `number`. This is also why you needed a cast here. Here's what I think this should be doing instead:

```diff
       let funcFrame = frameInfoByFunctionID.get(i32(lastLine.functionId))
-      if (lastLine.line && lastLine.line > 0) {
-        if (funcFrame != null) {
-          funcFrame.line = lastLine.line as number
-        }
+      const line = lastLine.line instanceof Long ? lastLine.line.toNumber() : lastLine.line
+      if (line && line > 0 && funcFrame != null) {
+        funcFrame.line = line
       }
```

If that looks right to you, can you please update the PR and run `npm run jest -- --updateSnapshot` to update the snapshot? After that, this should be good to land

Reply:
@jlfwong I tried that, but it is giving me a typescript error I don't understand when I try to run npm jest:

```
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/import/pprof.ts:85:45 - error TS2686: 'Long' refers to a UMD global, but the current file is a module. Consider adding an import instead.

    85       const line = lastLine.line instanceof Long ? lastLine.line.toNumber() : lastLine.line
                                                   ~~~~

```

Seems like some kind of a namespace issue? i guess `Long` isn't available by default to check with `instanceof`?

Reply:
ok, i was able to make it work by also adding an import:

```
import Long = require("long")
```

I've committed the results and pushed.